components/motor/roboclaw/roboclaw.go

```diff
@@ -4,21 +4,27 @@ package roboclaw
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"math"
 	"time"
 
 	"github.com/CPRT/roboclaw"
 	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
 	utils "go.viam.com/utils"
 
 	"go.viam.com/rdk/components/motor"
 	"go.viam.com/rdk/operation"
 	"go.viam.com/rdk/resource"
 )
 
-var model = resource.DefaultModelFamily.WithModel("roboclaw")
+var (
```

Comment:
[minor] can you add an extended note here with the steps to configure the roboclaw and any good URLs that point a user toward a useful website. This is also helpful for the docs team. Check out `components/motor/gpiostepper/gpiostepper.go` and `components/movementsensor/imuwit/imuwit.go` for examples.